commands/channelargs.go

```diff
@@ -56,3 +58,68 @@ func getChannelFromChannelArg(c client.Client, channelArg string) *model.Channel
 
 	return channel
 }
+
+func getChannelsFromArgs(c client.Client, channelArgs []string) ([]*model.Channel, error) {
+	var (
+		channels []*model.Channel
+		result   *multierror.Error
```

Comment:
We don't continue if we have an error in the call site right? Do we need to append multiple errors? We can just return if we got an error. We can also avoid another unnecessary dependency.

Reply:
Depends on the expectations. Personally, if I'm working with multiple parameters seems more useful to inform about the errors rather than halting the program.

Prior to this PR, when iterating through the users, if the user is null, we print an error and continue with the command.

About the dependency, I can create a basic `multierror` for this purpose

Reply:
Currently in our `getEntityFromEntityArgs` we're processing the results that we fetch correctly, so I think we should do the same thing in this case to avoid breaking compatibility with the current command behavior

Reply:
Sorry for the late reply @jecepeda!! My point was; if we return an error/errors from the `getChannelsFromArgs` do we continue in the call site? If we don't, there is no need to continue the operation here. But if we still continue in the call site, there will be another problem. Because I'd expect to halt my operation if I get an error from `getChannelsFromArgs` in the call site. Therefore I think either we don't return error at all, or return the error immediately.

Reply:
As discussed online, we can continue with this approach. @jecepeda could you add a comment to the function to make clear that function is returning values and errors at the same time? I mean let's make it more explicit.

Reply:
Perfect @isacikgoz . I just did it